docs/pages/inboxes/history-sync.md

```diff
@@ -0,0 +1,138 @@
+# Enable history sync for apps built with XMTP
+
+Enable the history sync feature to give your users a way to sync decrypted historical data from an existing app installation to a new app installation.
+
+This historical data includes:
+
+- Conversations
+- Conversation messages
+- Consent state
+
+History sync enables your users pick up conversations where they left off, regardless of the app installation they use. All they need is an existing app installation to provide the data.
```

Comment:
Do we want to say something specific about how this helps with lost devices? I have one device with Converse installed and device sync is enabled. I lose the device -- can I get a new device, install Converse and use device sync to restore my inbox?